pkg/render/fluentd.go

```diff
@@ -157,6 +167,8 @@ type FluentdConfiguration struct {
 
 	// Whether or not the cluster supports pod security policies.
 	UsePSP bool
+	// Whether to use User provided certificate or not.
+	UseUserCA bool
```

Comment:
Should be called UseUserCertificate, since we do not know if the user supplies a cert or a CA cert

Reply:
Updated